spec.html

```diff
@@ -34983,22 +34984,29 @@ <h1>Pattern</h1>
             1. Return _m_(_x_, _c_).
         </emu-alg>
         <emu-note>
-          <p>A Pattern evaluates (&ldquo;compiles&rdquo;) to an Abstract Closure value. RegExpBuiltinExec can then apply this procedure to a String and an offset within the String to determine whether the pattern would match starting at exactly that offset within the String, and, if it does match, what the values of the capturing parentheses would be. The algorithms in <emu-xref href="#sec-pattern-semantics"></emu-xref> are designed so that compiling a pattern may throw a *SyntaxError* exception; on the other hand, once the pattern is successfully compiled, applying the resulting Abstract Closure to find a match in a String cannot throw an exception (except for any implementation-defined exceptions that can occur anywhere such as out-of-memory).</p>
+          <p>A Pattern compiles to an Abstract Closure value. RegExpBuiltinExec can then apply this procedure to a String and an offset within the String to determine whether the pattern would match starting at exactly that offset within the String, and, if it does match, what the values of the capturing parentheses would be. The algorithms in <emu-xref href="#sec-pattern-semantics"></emu-xref> are designed so that compiling a pattern may throw a *SyntaxError* exception; on the other hand, once the pattern is successfully compiled, applying the resulting Abstract Closure to find a match in a String cannot throw an exception (except for any implementation-defined exceptions that can occur anywhere such as out-of-memory).</p>
         </emu-note>
       </emu-clause>
 
-      <emu-clause id="sec-disjunction">
-        <h1>Disjunction</h1>
-        <p>With parameter _direction_.</p>
-        <p>The production <emu-grammar>Disjunction :: Alternative</emu-grammar> evaluates as follows:</p>
-        <emu-alg>
-          1. Evaluate |Alternative| with argument _direction_ to obtain a Matcher _m_.
-          1. Return _m_.
-        </emu-alg>
-        <p>The production <emu-grammar>Disjunction :: Alternative `|` Disjunction</emu-grammar> evaluates as follows:</p>
+      <emu-clause id="sec-compilesubpattern" type="sdo" oldids="sec-disjunction,sec-alternative,sec-term">
+        <h1>
+          Runtime Semantics: CompileSubpattern (
+            _direction_: 1 or -1,
+          )
+        </h1>
+        <dl class="header">
+          <dt>description</dt>
+          <dd>It returns a Matcher.</dd>
+        </dl>
+        <emu-note>
+          <p>This section is amended in <emu-xref href="#sec-compilesubpattern-annexb"></emu-xref>.</p>
+        </emu-note>
+
+        <!-- Disjunction -->
```

Comment:
Did you mean to leave these comments in?

Reply:
Yup. In the status quo, each nonterminal gets its own clause, but in this PR, three of the new SDOs (CompileSubpattern, CompileAtom, and CompileToCharSet) include rules for multiple nonterminals, and I wasn't sure if the loss of a per-nonterminal heading would be 'disorienting', so I left these comments in case something was wanted.

Looking at the rendered spec, I don't really feel the lack of per-nonterminal headings, but I can imagine that some people might like them. Looking at the source, I think the comments help navigation slightly, but I wouldn't be crushed if the editors wanted them removed.